src/toil/test/__init__.py

```diff
@@ -48,6 +54,10 @@ class ToilTest(unittest.TestCase):
     _tempBaseDir = None
     _tempDirs = None
 
+    # Use us-west-2 unless running on EC2, in which case use the region in which
+    # the instance is located
+    AWSRegion = _region() if runningOnEC2() else 'us-west-2'
```

Comment:
I just found a downside to this approach. This code will throw NameErrors if the functions are not defined above their usage (_region, runningOnEC2, and functions used within those functions). I think this is because it's run at import time since it's class level and not protected by __init__...